__tests__/plugin.test.js

```diff
@@ -18,19 +18,20 @@ describe('babel plugin', () => {
   });
 
   it('injects its version', () => {
+    process.env.REANIMATED_PLUGIN_TESTS = 'jest version check';
```

Comment:
Instead of using a different string (i.e. `'jest version check'` vs `'jest'`) let's just remove this env variable for this test:
```suggestion
    delete process.env.REANIMATED_PLUGIN_TESTS;
```
If this doesn't work, we can just assign `undefined`.